Review of the documentation (#105)
open-plan-tool/documentation: docs/getting_started/quick_start_guide.rst

======================
Quick start
======================

First steps
------------

CREATE AN ACCOUNT
^^^^^^^^^^

> The first step to start using open_plan is to create an account. To do this, you must click on the "Sign up" button at the top right corner and fill in the following information: your name, email address, username and password. Finally, do not forget to read and accept the privacy statement. You should receive an email with a link to confirm the account creation. 

Author: ```suggestion
The first step to start using open_plan is to create an account. To do this, you must click on the "Sign up" button at the top right corner and fill in the following information: your name, email address, username and password. Finally, do not forget to read and accept the privacy statement. You should receive an email with a link to confirm the account creation. You can find our privacy policy here: xxxx
```

Author: Add link

Author: @Bachibouzouk please add the link to the privacy policy

.. image:: ../images/IMG1_Sign_up.png
 :width: 100%

EXPLORE THE DASHBOARD
^^^^^^^^^^

When you log in, the following dashboard is displayed. In the following image the points described below are labeled with numbers and letters.

.. image:: ../images/IMG2_Main_Screen.png
 :width: 100%

> **1. Logo. Clicking here will take you back to the home screen.**

Author: ```suggestion
**1. Logo. Clicking here will take you back to the home screen and your projects. Here you find all the projects you have created, and you can create new projects.**
```

> **2. Top navigation menu. Located on the upper right part of the screen, here you will find the shortcuts to:**

Reviewer: We still need, right @c-moeller ?

>     a) Simulation results. Visualize the results of your scenarios in this tab.

Author: ```suggestion
```

>     b) New scenario. Your projects can be composed of multiple scenarios and in each one you can define different parameters. Here you can create them. 

Author: ```suggestion
```

>     c) My projects. Here you find all the projects you have created, and you can create new projects. 

Author: ```suggestion
```

>     d) Documentation. Key information so that you can get to know the tool and develop your projects.

Author: ```suggestion
    a) Documentation. Key information so that you can get to know the tool and develop your projects.
```

>     e) Change the language. English (en) and German (de) are supported.

Author: ```suggestion
    b) Use cases. Here you will find some use cases where the functionalities of the system and its application in different projects and scenarios are presented.
    c) We are happy to hear about your experience with open_plan, so feel free to share your questions, comments and suggestions here.
    d) Change the language. English (en) and German (de) are supported.
```

>     f) Profile drop-down options.

Author: ```suggestion
    e) Profile drop-down options.
```

**3. Dashboard. Here is the link to create a new project or the list of current projects when they already exist.**

**4. Bottom navigation menu. Located at the bottom right of the screen. Here you will find:**
    g) Contact. General information about the project, a section where you can leave your feedback and find the link to GitHub, where you can follow the development or find useful information related to how to host the tool by yourself.

>     h) Documentation. Key information so that you can get to know the tool and develop your projects.

Author: ```suggestion
```

>     i) Use cases. Here you will find some cases where the functionalities of the system and its application in different projects and scenarios are presented.

Author: ```suggestion
```

2) Energy system design

In this section the energy system will be designed using different components, which are located on the left panel: Providers, Production, Conversion, Storage, Demand and Bus. 
In the graphic panel drag the components you need to design your energy system, do not forget to include the buses. Note that assets must be connected to each other using a bus, and that interconnecting buses is not allowed. Connect the components together using the green and red terminals. The green terminals represent inputs, while the red terminals represent outputs, see the following example:

.. image:: ../images/IMG_example_connections.png
 :width: 100%

The components representing battery energy storage systems (BESS) have been defined with one input and one output. The BESS can be connected directly to the electrical bus; please note that the bus is supplied and feeds the battery at the same time.

> .. image:: ../images/IMG_example_storage.png

Author: ```suggestion

.. image:: ../images/IMG_example_storage.png
```

 :width: 100%

> 

Reviewer 2: image not shown, only code to include it

Author: Hi @NargesChinichian did you fix it somewhere? I don't see it. I will insert a break line. You can reject the suggestion if you have fixed it already somewhere else.

.. image:: ../images/IMG_setup_Bus.png
 :width: 100%

.. image:: ../images/IMG_setup_PV.png
 :width: 100%

.. image:: ../images/IMG_setup_HeatDemand.png
 :width: 100%

Before proceeding to the next section, be sure to complete the information requirements for each component and save the energy system.

3) Constraints

Now we must decide whether or not to activate the constraints. To do this, there is a drop-down menu in each case. The two constraints that can be defined are: 

	- **Minimal degree of autonomy**. It refers to the definition of a lower boundary on the degree of autonomy of the energy system. This factor can take values between 0 and 1, with the value close to zero showing a degree of autonomy with high dependence on the energy supplier (energy coming from the grid), while a degree of autonomy of 1 represents an autonomous energy system.

	- **Minimal share of renewables**. This constraint defines a lower boundary for the renewable share of the system, where both local generation as well as the renewable share of energy providers are taken into account. 
	
The above constraints apply to the entire system, but not to specific sectors individually.

> 

Reviewer 2: After setting up my scenario, components and constraints clicking on "next" or "simulation" leads to an Server Error (s. screenshots)
![OP_ss1](https://github.com/open-plan-tool/documentation/assets/59662856/6149cfb7-4d0f-4622-ba2e-99d09e38e99f)
![OP_ss2](https://github.com/open-plan-tool/documentation/assets/59662856/db843eea-d431-472c-92ae-795babc23efd)

```

    g) Contact. General information about the project, a section where you can leave your feedback and find the link to GitHub, where you can follow the development or find useful information related to how to host the tool by yourself.
    h) Documentation. Key information so that you can get to know the tool and develop your projects.
    i) Use cases. Here you will find some cases where the functionalities of the system and its application in different projects and scenarios are presented.
    j) License. You can find the license information here.
    k) Imprint. Important information about the tool and some disclaimers are in this section.
    l) Privacy. Information on data protection, data processing and the legal and policy issues around them.

CREATE A PROJECT
^^^^^^^^^^

To create a project in open_plan, the user has three options:

.. image:: ../images/New_project_EN.png
 :width: 30%

:Empty project:
Here a project is created from scratch. When you select this option you must configure your project and for that you must include the following information: 

- Name

- Description

- Country

- Location (coordinates)

- Duration (years)

- Currency

- Discount factor

> - Tax factor 

Author: I think we can delete the tax factor from the tool, right? @Bachibouzouk

> - Tax factor 

Author: I have deleted the tax factor from the documentation

.. image:: ../images/IMG_4_New_scenario.png
 :width: 50%

When creating a scenario from scratch, there are four steps to go through; 
1) Scenario setup, 2) Energy system design, 3) Constraints and 4) Simulation.  At the top you will see the name of the project in bold type, the name of the scenario, the four steps for scenario creation and identify which step you are in.

Maybe show the 4 steps (the top line with the 4 scenario steps) in a figure

.. image:: ../images/IMG_Scenario_Steps.png
 :width: 100%

Below is a brief description of what should be done at each step.

1) Scenario setup

The setup consists of assigning a name to the scenario, its corresponding description, the evaluated period or the number of days the simulation will run, the length of the time steps of the simulation in minutes, the start date (keep in mind that this date is important for getting the data from the time series and for plotting the data) and the fixed project costs, where the planning and development costs are included.

> 

Reviewer 2: the fixed project costs appear more than once, e.g. as well at the component configuration. A remark that if they're already included in the scenario setup, you can stay with zero for the component configurations

Reviewer: There are fixed *project* costs and fixed costs for each assets, the first one is a cost that occurs only once within the whole project and the second one can be different for each asset and also are multiplied by how many asset units are being optimized. Maybe with this extra explanation it would be easier to understand?

Reviewer 2: yes, adding this to the doc or the infoboxes in OP would make it clear

.. image:: ../images/IMG_example_connections.png
 :width: 100%

The components representing battery energy storage systems (BESS) have been defined with one input and one output. The BESS can be connected directly to the electrical bus; please note that the bus is supplied and feeds the battery at the same time.
.. image:: ../images/IMG_example_storage.png
 :width: 100%

When you click on the components, a screen appears where you can configure the different parameters. Each component is different, however, you will typically find three ways to complete the information: spaces to enter values, drop-down lists with default information or buttons to load time series (in this case, a graph will be displayed where the loaded data series can be previewed). Below we show you as an example some of the component setup screens, where you will have to include the parameters.

.. image:: ../images/IMG_setup_Bus.png
 :width: 100%

.. image:: ../images/IMG_setup_PV.png
 :width: 100%

.. image:: ../images/IMG_setup_HeatDemand.png
 :width: 100%

Before proceeding to the next section, be sure to complete the information requirements for each component and save the energy system.

> 

Reviewer: I agree with you this information lacks
